docs/Getting Started/openSUSE/openSUSE Leap Root on ZFS.rst

```diff
@@ -406,6 +407,22 @@ Step 2: Disk Formatting
    - The pool name is arbitrary. If changed, the new name must be used
      consistently. On systems that can automatically install to ZFS, the root
      pool is named ``rpool`` by default.
+   - If you want to use grub bootloader, you must to set::
```

Comment:
It's not obvious, user already created root pool. Let's move it into root pool creation before actual command. Grub is usual choice, so it would be great to add it in command for user and add notes why user may want to enable some features if grub won't be used.